### PR TITLE
update Bernoulli example to use logits, not p

### PR DESCRIPTION
**Pull request by rdipietro**
I couldn't find the examples that correspond directly to the website. If you want me to update any other files, just let me know.

### CODE REVIEW

**Comment by dustinvtran** [comment]
Thanks! Could you collapse the `logits` line and place it into the Bernoulli instantiation directly?

**Comment by rdipietro** [comment]
Sure, all set.
